templates/zerver/help/add-a-bot-or-integration.md

```diff
@@ -7,8 +7,9 @@ Bots and integrations are features that are accessible to all members of an orga
 {!go-to-the.md!} [Your bots](/#settings/your-bots)
 {!settings.md!}
 
-2. On this page, under the **Add a new bot** view, you can enter a name, an email, and
-optionally an avatar for your bot. Then click the **Create bot** button to create your bot.
+2. On this page, under the **Add a new bot** view, you can specify the bot type,
+   enter a name, an email, and optionally an avatar for your bot.
```

Comment:
This reads kind of awkwardly imo. Something like `specify the bot type, a name, an email, and optionally an avatar...` perhaps?